databricks/koalas/window.py

```diff
@@ -51,6 +54,16 @@ def _apply_as_series_or_frame(self, func):
     def count(self):
         def count(scol):
             return F.count(scol).over(self._window)
+
+        if LooseVersion(pd.__version__) >= LooseVersion('1.0.0'):
```

Comment:
@itholic, given our discussion, we should match it to the latest behaviour. I think we don't have to check the pandas version.